params/config.go

```diff
@@ -647,6 +757,27 @@ func (c *ChainConfig) CheckConfigForkOrder() error {
 			lastFork = cur
 		}
 	}
+
+	// Check that all forks with blobs explicitly define the blob schedule
+	// configuration.
+	bsc := c.BlobScheduleConfig
+	if bsc == nil {
+		bsc = &BlobScheduleConfig{}
+	}
+	for _, cur := range []struct {
+		name      string
+		timestamp *uint64
+		config    *BlobConfig
+	}{
+		{name: "cancun", timestamp: c.CancunTime, config: bsc.Cancun},
+		{name: "prague", timestamp: c.PragueTime, config: bsc.Prague},
+		{name: "verkle", timestamp: c.VerkleTime, config: bsc.Verkle},
+	} {
+		if cur.timestamp != nil && cur.config == nil {
+			return fmt.Errorf("unsupported fork configuration: missing blob configuration entry for %v in schedule", cur.name)
```

Comment:
EIP:

`When there is no explicit configuration for the current fork, use the last
specified fork value. `

It means unconfigured blob is accepted and the preceding one should be used in case

Reply:
This was actually removed in a later version of the EIP: https://github.com/ethereum/EIPs/pull/9240